src/assets/js/react/components/FontManager/SearchBox.js

```diff
@@ -120,17 +120,19 @@ export class SearchBox extends Component {
    */
   render () {
     return (
-      <input
-        type='search'
-        id='font-manager-search-box'
-        className='wp-filter-search'
-        placeholder={GFPDF.fontManagerSearchPlaceHolder}
-        value={this.state.searchInput}
-        onChange={this.handleSearch}
-        onKeyDown={e => e.keyCode === 13 && e.preventDefault()}
-        ref={node => (this.input = node)}
-        tabIndex='143'
-      />
+      <div role='form'>
```

Comment:
Couple comments on this:

1. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Form_Role recommends using `<form>` where possible (but doesn't go into why you would choose to use `role="form"` instead of `<form>`). What research have you done into use the role-base alternative, over using the native `<form>` tag?

2. Regardless of which way we go, shouldn't we look at wrapping both the search and results in a FORM element, since the results include form-related actions (delete for example)?

Reply:
Also, it looks like `<form role="search">` is preferred https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Search_role